odoo/tools/misc.py

```diff
@@ -767,7 +767,7 @@ def __missing__(self, key):
         return unquote(key)
 
 
-class mute_logger(object):
+class mute_logger(logging.Handler):
```

Comment:
Why did you change this?
I don't understand.

Reply:
I gave a quick explanation about that at the end of the in the pr message but I either don't remember the precise corner case or it's not usefull anymore...

```python
    with lower_logging(25):
        with assertLogs(level='ERROR'):
            logger.error()  
```
In short, to work with assertLogs, lower_logging must be an handler. And during my testing I needed to change mute_logger to and handler too for some reason, but I'll try to remember why exactly or revert this change.